TERMS.rst

```diff
@@ -0,0 +1,7 @@
+===============
+Terms of use
+===============
+
+Data
+==========
+- Be very careful about including sensitive data in a repository. Highly sensitive data, such as personally identifiable information, **should not be included under any circumstances**. Other non-public data should not be included in repositories that are destined to be made public. If in doubt, leave it out.
```

Comment:
Maybe something like: "Any non-public data that gets merged into the main branch of the repository must be excised from the repository's history using e.g. `git-filter-branch`, BFG, or other similar methods."

Could consider another term of use around review policies, protected branches, and such like.